### PR TITLE
Add / fix docstrings

### PR DESCRIPTION
**Pull request by therealsilence**
- Add docstrings to inet.py
- Update dns.py, files.py with docstring standards
   - ro, req, defval params at end of first prop line
   - docstrings on second prop line

### CODE REVIEW

File: synapse/models/inet.py [hunk 1]
```diff
@@ -695,487 +695,791 @@ def _revModl201710111553(self):
...
+                        'doc': 'A related URL specified by the account (e.g., a personal or company web '
+                             'page, blog, etc.).'}),
+                    ('loc', {'ptype': 'str:lwr',
+                        'doc': 'A self-declared location for the account.'}),
+                    ('occupation', {'ptype': 'str:lwr',
+                        'doc': 'A self-declared occupation for the account.'}),
+                    ('dob', {'ptype': 'time',
+                        'doc': 'A self-declared date of birth for the account (if the account belongs to a person).'}),
                     # ('gender',{'ptype':'inet:fqdn','ro':1}),
+                    # ('bio:bt',{'ptype':'wtf','doc':'The web account's self documented blood type'}),
```
Comment (vertexmc) [inline synapse/models/inet.py]: Should we just remove these two commented out lines?

File: synapse/models/inet.py [hunk 2]
```diff
@@ -695,487 +695,791 @@ def _revModl201710111553(self):
...
-                    ('logout', {'ptype': 'time', 'doc': 'The time the account logged out of the service.'})
+                    ('acct', {'ptype': 'inet:web:acct', 'ro': 1, 'req': 1,
+                        'doc': 'The web account associated with the logon event.'}),
+                    ('acct:site', {'ptype': 'inet:fqdn', 'ro': 1,
+                        'doc': 'The site or service associated with the account.'}),
+                    ('acct:user', {'ptype': 'inet:user', 'ro': 1,
+                        'doc': 'The unique identifier for the account.'}),
+                    ('time', {'ptype': 'time',
+                        'doc': 'The date and time the account logged into the service.'}),
+                    ('ipv4', {'ptype': 'inet:ipv4',\
```
Comment (vertexmc) [inline synapse/models/inet.py]: Trailing `\`

File: synapse/models/inet.py (hunk 1 above)
Comment (invisig0th) [reply synapse/models/inet.py]: nah, lets leave these reminders for now.

File: synapse/models/inet.py (hunk 2 above)
Comment (invisig0th) [reply synapse/models/inet.py]: yea, lets remove the trailing \  as it's not needed

**Comment by vEpiphyte** [comment]
:heart: these

File: synapse/models/inet.py (hunk 2 above)
Comment (therealsilence) [reply synapse/models/inet.py]: fixed

**Pull request merged**